strongbox-storage/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactSearchServiceImplTest.java

```diff
@@ -69,6 +70,7 @@ public void removeRepositories()
     }
 
     @Test
+    @Ignore
```

Comment:
@sbespalov : Is this now broken? Is it because of the index type issue I pointed out earlier in my review?